src/tests/ftest/util/configuration_utils.py

```diff
@@ -493,6 +493,7 @@ def get(self, key, path=None, default=None):
                 Defaults to None.
         """
         # Convert a glob-style path into regular expression
+        re_path = ""
```

Comment:
If you need to make changes again, this line is not needed as the if/elif/else sets the value.

Reply:
The required changes in this file will be resolved by the landing of #1327